README.md

```diff
@@ -1,24 +1,11 @@
 # hyper-content-for
 
-Allows you to store HTML for use (and re-use) later.
```

Comment:
:+1: for a small summary. I just didn't get what "storing HTML for use (and re-use) later meant" and I couldn't come up with anything better, so I removed it. Hehe.